tools/types.ts

```diff
@@ -8,4 +8,4 @@ export enum SortOrder {
 }
 
 // Sort options type that contains an array of tuples of the field being sorted and the order
-export type SortOptions = ReadonlyArray<[string, SortOrder]>;
+export type SortOptions = Array<[string, SortOrder]>;
```

Comment:
Same here, wondering why this was changed to `Array`?

Reply:
afaik gql doesn't have readonly types so when trying to assign this to the order argument in the filter whose type is
`Maybe<Array<Array<Scalars['String']>>>`
 typescript would complain